core/templates/services/UpgradedServices.ts

```diff
@@ -915,7 +915,9 @@ export class UpgradedServices {
         upgradedServices['CamelCaseToHyphensPipe']);
     upgradedServices['ExtractImageFilenamesFromStateService'] =
       new ExtractImageFilenamesFromStateService(
-        upgradedServices['HtmlEscaperService']);
+        upgradedServices['HtmlEscaperService'],
+        upgradedServices['contentTranslationManagerService'],
+        upgradedServices['contentTranslationLanguageService']);
```

Comment:
These values are undefined. Why is this change required?

Reply:
Oops! I added this after I got a typescript check error. Hm, the functionality seemed to work fine before this change, so I wonder, do we really need an entry for "ExtractImageFilenamesFromStateService" here?

Reply:
Maybe we can remove this entry from here and also App.ts. In case any tests fail, you can `importAllAngularServices()` in the describe block. WDYT?